src/nle.c

```diff
@@ -31,6 +34,12 @@ init_nle(FILE *ttyrec)
     assert(ttyrec != NULL);
     nle->ttyrec = ttyrec;
 
+#ifdef NLE_BZ2_TTYRECS
+    int bzerror;
+    nle->ttyrec_bz2 = BZ2_bzWriteOpen(&bzerror, ttyrec, 9, 0, 0);
```

Comment:
What's the 9?

Reply:
The block size. 9 is the highest and also what alt.org uses.

http://www.cs.cmu.edu/afs/cs/project/pscico-guyb/realworld/99/code/bzip2-0.9.5c/manual_3.html#SEC30